lib/all-badge-examples.js

```diff
@@ -2046,15 +2046,15 @@ const allBadgeExamples = [
         previewUri: '/issuestats/i/github/expressjs/express.svg'
       },
       {
-        title: '(long form)',
+        title: 'Issue Stats (long form)',
         exampleUri: '/issuestats/i/long/github/expressjs/express.svg'
       },
       {
-        title: 'Issue Stats',
+        title: 'Pull Request Stats',
         previewUri: '/issuestats/p/github/expressjs/express.svg'
       },
       {
-        title: '(long form)',
+        title: 'Pull Request Stats (long form)',
         exampleUri: '/issuestats/p/long/github/expressjs/express.svg'
```

Comment:
I changed the title here to match what the badge is showing. 

Example badge shows: `pull closure | 16 hrs`